pcsx2/gui/ConsoleLogger.cpp

```diff
@@ -425,7 +458,8 @@ ConsoleLogFrame::ConsoleLogFrame( MainEmuFrame *parent, const wxString& title, A
 
 	menuFontSizes.AppendSeparator();
 	menuFontSizes.Append( MenuId_ColorScheme_Light,	_("&Light theme"),	_t("Default soft-tone color scheme."), wxITEM_RADIO );
-	menuFontSizes.Append( MenuId_ColorScheme_Dark,	_("&Dark theme"),	_t("Classic black color scheme for people who enjoy having text seared into their optic nerves."), wxITEM_RADIO );
+	menuFontSizes.Append( MenuId_ColorScheme_Dark,	_("&Dark theme"),	_t("Modern twist on Dark Theme which does not burn optic nerves."), wxITEM_RADIO );
+	menuFontSizes.Append( MenuId_ColorScheme_ClassicDark,	_("&Classic Dark theme"),	_t("Classic black color scheme for people who enjoy having text seared into their optic nerves."), wxITEM_RADIO );
```

Comment:
Seemed to have deleted my comment accidentally.  Inappropriate descriptions. Please use something like "Modern dark theme" and "Classic PCSX2 style dark theme"